src/main/java/com/google/sps/data/HttpServletUtils.java

```diff
@@ -37,4 +54,80 @@ public static String getParameter(HttpServletRequest request, String name, Strin
     }
     return value;
   }
+
+  /** Returns a BlobKey for the uploaded file, or null if the user didn't upload a file. */
```

Comment:
Nit: Please include JavaDoc (e.g. @param...) for your functions.

Reply:
Done!